### PR TITLE
Fix Page links

### PR DESCRIPTION
**Pull request by narendracode**
## Proposed changes

In the following pages, link is wrong.

- README.md
- TERMS-OF-USE.md
- packages/caver-rpc/src/klay.js

changes :
- https://docs.klaytn.com/bapp
+ https://docs.klaytn.com/dapp
## Types of changes

Please put an x in the boxes related to your change.

- [x ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [ x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [ x] Lint and unit tests pass locally with my changes
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] I have added necessary documentation (if appropriate)
- [ x] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.
https://github.com/klaytn/caver-js/issues/579
## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/caver-rpc/src/klay.js [hunk 1]
```diff
@@ -2147,7 +2147,7 @@ class Klay {
                     logs: {
                         params: 1,
                         inputFormatter: [formatters.inputLogFormatter],
-                        subscriptionHandler: function(output) {
+                        subscriptionHandler: function (output) {
```
Comment (jimni1222) [inline packages/caver-rpc/src/klay.js:2150]: Can you remove those modifications?
You can run `npm run lintFix` to fix linting.

Comment (narendracode) [reply packages/caver-rpc/src/klay.js:2150]: alright let me do that @jimni1222

Comment (narendracode) [reply packages/caver-rpc/src/klay.js]: done @jimni1222

**Review by jimni1222 — APPROVED** [review]
LGTM =) Thank you for your contribution.
I will check CI process for this PR, and when CI is done i will merge it.

**Review by aeharvlee — APPROVED** [review]
Thanks for your contribution :)

**Comment by iv0rish** [comment]
Hi @narendracode, I'm checking CircleCI and it seems having some problem. 

Do you use CircleCI with your Github account? We have reported some similar issues about duplicated pipeline executed if contributors have personal CircleCI account linked with Github. Please kindly let me know and I will discuss about it more 👍

**Comment by narendracode** [comment]
> Hi @narendracode, I'm checking CircleCI and it seems having some problem.
> 
> Do you use CircleCI with your Github account? We have reported some similar issues about duplicated pipeline executed if contributors have personal CircleCI account linked with Github. Please kindly let me know and I will discuss about it more 👍

@iv0rish I don't use circleCI. Not sure what is the issue :)

**Comment by narendracode** [comment]
> Hi @narendracode, I'm checking CircleCI and it seems having some problem.
> 
> Do you use CircleCI with your Github account? We have reported some similar issues about duplicated pipeline executed if contributors have personal CircleCI account linked with Github. Please kindly let me know and I will discuss about it more 👍

@iv0rish  I do have TravisCI though, do you think that could be an issue here?

**Review by sirano11 — APPROVED** [review]

**Comment by iv0rish** [comment]
> > Hi @narendracode, I'm checking CircleCI and it seems having some problem.
> > Do you use CircleCI with your Github account? We have reported some similar issues about duplicated pipeline executed if contributors have personal CircleCI account linked with Github. Please kindly let me know and I will discuss about it more 👍
> 
> @iv0rish I do have TravisCI though, do you think that could be an issue here?

Thanks @narendracode, TravisCI won't effect to CircleCI. 
For Klaytn repos, Github triggers CircleCI Webhook when new PR opens. CircleCI responds HTTP 400 error to your PR, even latest PR from dependabot has finished CI pipeline.. Can you try some other steps below?

1. Could you add some new commits to this PR? New commit will trigger CircleCI pipeline.
2. If pipeline still does not working, could you please close & re-open this PR?

I apologize for any inconvenience this may cause 🙏

**Comment by narendracode** [comment]
> > > Hi @narendracode, I'm checking CircleCI and it seems having some problem.
> > > Do you use CircleCI with your Github account? We have reported some similar issues about duplicated pipeline executed if contributors have personal CircleCI account linked with Github. Please kindly let me know and I will discuss about it more 👍
> > 
> > 
> > @iv0rish I do have TravisCI though, do you think that could be an issue here?
> 
> Thanks @narendracode, TravisCI won't effect to CircleCI. For Klaytn repos, Github triggers CircleCI Webhook when new PR opens. CircleCI responds HTTP 400 error to your PR, even latest PR from dependabot has finished CI pipeline.. Can you try some other steps below?
> 
> 1. Could you add some new commits to this PR? New commit will trigger CircleCI pipeline.
> 2. If pipeline still does not working, could you please close & re-open this PR?
> 
> I apologize for any inconvenience this may cause 🙏

Alright @iv0rish   let me try those steps you mentioned.

**Comment by narendracode** [comment]
@iv0rish  you may check ci logs i just pushed.

**Comment by iv0rish** [comment]
> @iv0rish you may check ci logs i just pushed.

@narendracode Thanks, but still expecting Invalid response from CircleCI. I think we should move to next step: re-open the PR.

**Comment by iv0rish** [comment]
@narendracode Could you please leave this PR(not closing), make a new branch with same commits and open PR again?

**Comment by narendracode** [comment]
@iv0rish refer to https://github.com/klaytn/caver-js/pull/584

**Comment by iv0rish** [comment]
@narendracode Still getting same error, I think we need to investigate this deeply and maybe escalate to CircleCI. I will keep update the progress here.

**Comment by narendracode** [comment]
@iv0rish sure no worries. Happy to help with anything.

**Comment by narendracode** [comment]
Just checking if all is okay here with PR and repo settings? :) @iv0rish

**Comment by iv0rish** [comment]
> Just checking if all is okay here with PR and repo settings? :) @iv0rish

Nothing wrong with PR or Github settings, we're looking into this with CircleCI. Apologize for delays during lunar new year.

**Comment by iv0rish** [comment]
@narendracode Thank you for your patience - Finally CI works and your PR can be merged!

For the reason we heard from CircleCI is: There is credential errors on your CircleCI account, it happens sometime when you does not sign in to CircleCI for a while. They have cleared credential cache from your account and now it works.

cc. @jimni1222 @aeharvlee @sirano11

**Comment by narendracode** [comment]
> @narendracode Thank you for your patience - Finally CI works and your PR can be merged!
> 
> For the reason we heard from CircleCI is: There is credential errors on your CircleCI account, it happens sometime when you does not sign in to CircleCI for a while. They have cleared credential cache from your account and now it works.
> 
> cc. @jimni1222 @aeharvlee @sirano11

Thank you @iv0rish That's great it works now.

**Comment by jimni1222** [comment]
@narendracode I merged this PR :) Thank you for your contribution again!

**Pull request merged**